odoo/addons/base/ir/ir_translation.py

```diff
@@ -257,27 +257,21 @@ def _auto_init(self):
         res = super(IrTranslation, self)._auto_init()
         cr = self._cr
 
-        cr.execute("SELECT indexname FROM pg_indexes WHERE indexname LIKE 'ir_translation_%'")
-        indexes = [row[0] for row in cr.fetchall()]
-
         # Removed because there is a size limit on btree indexed values (problem with column src):
         # cr.execute('CREATE INDEX ir_translation_ltns ON ir_translation (name, lang, type, src)')
         # cr.execute('CREATE INDEX ir_translation_lts ON ir_translation (lang, type, src)')
         #
         # Removed because hash indexes are not compatible with postgres streaming replication:
         # cr.execute('CREATE INDEX ir_translation_src_hash_idx ON ir_translation USING hash (src)')
```

Comment:
Do we still need these commented create indexes? Just note why we're dropping the indices